eliot/prettyprint.py

```diff
@@ -0,0 +1,73 @@
+"""
+API and command-line support for human-readable Eliot messages.
+"""
+
+from __future__ import unicode_literals
+
+from datetime import datetime
+from sys import stdin, stdout, argv
+from json import loads
+
+from six import text_type as unicode, PY2
+
+
+def pretty_print(message):
+    """
+    Convert a message dictionary into a human-readable string.
+
+    @param message: Message to parse, as dictionary.
```

Comment:
I guess the only issue that is not a public API yet, whereas this is going to be. So maybe later add an additional method that takes a `WrittenMessage`?